spec/models/service_spec.rb

```diff
@@ -47,6 +47,46 @@
       subject.name = "Apply for a license to do something"
     end
 
+    describe '#deploy_key' do
+      context 'when nil' do
+        it 'is valid' do
+          subject.valid?
+          expect(subject.errors[:deploy_key]).to be_blank
+        end
+      end
+
+      context 'when empty string' do
+        it 'is valid' do
+          subject.valid?
+          expect(subject.errors[:deploy_key]).to be_blank
```

Comment:
Is this actually testing an empty string? Looks exactly the same as the `"when nil"` spec above.

Reply:
good spot. i'm now testing nil and empty string